internal/sessions/cookie_store.go

```diff
@@ -10,47 +10,39 @@ import (
 	"github.com/pomerium/pomerium/internal/cryptutil"
 )
 
-// ChunkedCanaryByte is the byte value used as a canary prefix to distinguish if
```

Comment:
Changes in `cookie_store` are almost all rearranging of methods / reindentation. :rage2: